src/docs/patterns/overlay.mdx

```diff
@@ -1,73 +1,132 @@
 
-Sidebars are also used extensively for displaying information about selected items in the main interface (e.g., selecting an item in a list would open a sidebar on the right with additional details about the selected item). They can also present simple form data and inputs (such as radio buttons, checkbox, dropdown menus, etc.) for editing a selected item. This type of sidebar should be temporary such that the user can easily close it with a close button or clicking elsewhere on the screen.
+Sidebars (Side Sheets in the Material documentation) are used to present supplemental information to users. Most commonly, they display information about selected items in the main interface (e.g., selecting an item in a list would open a sidebar on the right with additional details about the selected item).
 
-### Modal vs. Sidebar
+They can also present simple form data and inputs (such as radio buttons, checkbox, dropdown menus, etc.) for editing a selected item. This type of sidebar should be temporary such that the user can easily close it with a close button or clicking elsewhere on the screen.
 
-In general, modals and sidebars can be used together to set up and display complex data. When building your application, you may be faced with the question of "Should I use a modal or a sidebar for this?" In general, modals should be used when creating new data, and sidebars should be used for viewing and editing existing data. In some circumstances, you may need a full-page overlay (sidebar or bottomsheet) to accomplish more complex tasks.
+Sidebars should always appear on the opposite side of the screen as the main [navigation](/patterns/navigation).
 
-## Bottom Sheet
+> In general, modals should be used when creating new data, and sidebars should be used for viewing and editing existing data.
 
-A bottom sheet is similar to a modal dialog but slides in from the bottom of the screen. A dark overlay is usually used to block user interaction with the main app until the bottom sheet is closed.
+## Bottom Sheet
 
 <MaterialDesignDescription title={'Sheets: Bottom'} url={'https://material.io/components/sheets-bottom/'} />
 
-This is a very common pattern in mobile applications but can also be used in responsive web apps. Typically, the bottom sheet contains global actions for the entire page and is activated by a button in the app bar.
+Bottom sheets are content surfaces that slide in from the bottom of the screen, primarily used in mobile applications. There are three types: standard, modal, and expandable. The content shown in a bottom sheet can be as simple as a list of items or as complex as a full screen layout.
 
-<ImageGrid images={[BasicBottomsheet, ComplexBottomsheet]} caption={'A Bottom Sheet can be simple or complex.'} />
+### Standard
 
-### Basic
+A standard bottom sheet is always visible and non-blocking, used most commonly for multitasking, such as displaying media player controls while browsing other content.
```

Comment:
I don't recall if we have any media-player-alike pattern in our designs. If a design doesn't exist and its contents are already covered by material, I am inclined to not mention it.